Add BeagleV-Fire overlay support to Makefile

This patch adds mpfs_beaglev_fire to the supported platforms list and
includes the necessary build rules for BeagleV-Fire device tree overlays.

Signed-off-by: Community Contributor <community@example.com>

--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@
 KERNEL_DIR?=../linux
 KERNEL_BUILD_DIR?=$(shell realpath --relative-to=. $(KERNEL_DIR))
 DTC?=$(KERNEL_BUILD_DIR)/scripts/dtc/dtc
-BDIR?=sam9x60ek sama5d29_curiosity sama5d27_som1_ek sama5d27_wlsom1_ek sama5d2_icp sama5d2_ptc_ek sama5d2_xplained sama5d2_xplained_grts sama5d3_xplained sama5d3_eds sama5d4_xplained sama7d65_curiosity sama7g5ek sama7g54_curiosity sam9x60_curiosity sam9x75eb mpfs_icicle mpfs_icicle_amp mpfs_video pic64gx_curiosity_kit
+BDIR?=sam9x60ek sama5d29_curiosity sama5d27_som1_ek sama5d27_wlsom1_ek sama5d2_icp sama5d2_ptc_ek sama5d2_xplained sama5d2_xplained_grts sama5d3_xplained sama5d3_eds sama5d4_xplained sama7d65_curiosity sama7g5ek sama7g54_curiosity sam9x60_curiosity sam9x75eb mpfs_icicle mpfs_icicle_amp mpfs_beaglv_fire mpfs_video pic64gx_curiosity_kit
 
 # workaround to make mkimage use the same dtc as we do
 PATH:=$(shell dirname $(DTC)):$(PATH)
@@ -13,6 +13,7 @@
 AT91SAM9X5EK_DTBO_OBJECTS:= $(patsubst %.dtso,%.dtbo,$(wildcard at91sam9x5ek/*.dtso))
 MPFS_ICICLE_DTBO_OBJECTS:= $(patsubst %.dtso,%.dtbo,$(wildcard mpfs_icicle/*.dtso))
 MPFS_ICICLE_AMP_DTBO_OBJECTS:= $(patsubst %.dtso,%.dtbo,$(wildcard mpfs_icicle_amp/*.dtso))
+MPFS_BEAGLV_FIRE_DTBO_OBJECTS:= $(patsubst %.dtso,%.dtbo,$(wildcard mpfs_beaglv_fire/*.dtso))
 MPFS_VIDEO_DTBO_OBJECTS:= $(patsubst %.dtso,%.dtbo,$(wildcard mpfs_video/*.dtso))
 MPFS_DISCOVERY_KIT_DTBO_OBJECTS:= $(patsubst %.dtso,%.dtbo,$(wildcard mpfs_discovery_kit/*.dtso))
 PIC64GX_CURIOSITY_KIT_DTBO_OBJECTS:= $(patsubst %.dtso,%.dtbo,$(wildcard pic64gx_curiosity_kit/*.dtso))
@@ -43,6 +44,8 @@
 
 mpfs_icicle_amp_dtbos: $(MPFS_ICICLE_AMP_DTBO_OBJECTS)
 
+mpfs_beaglv_fire_dtbos: $(MPFS_BEAGLV_FIRE_DTBO_OBJECTS)
+
 mpfs_video_dtbos: $(MPFS_VIDEO_DTBO_OBJECTS)
 
 mpfs_discovery_kit_dtbos: $(MPFS_DISCOVERY_KIT_DTBO_OBJECTS)